src/Uno.UI/DataBinding/BindingPath.cs

```diff
@@ -478,15 +478,37 @@ internal object GetSubstituteValue()
 				return GetSourceValue(_substituteValueGetter);
 			}
 
+			private bool _isDataContextChanging;
+
 			private void OnDataContextChanged()
 			{
 				if (DataContext != null)
 				{
 					ClearCachedGetters();
+					if (_propertyChanged.Disposable != null)
+					{
+						try
```

Comment:
@jeromelaban Should I remove this on Wasm?

Reply:
Yes, that will most probably help for performance.